backend/tag/usecase/src/interactor/get_tag.rs

```diff
@@ -80,28 +90,48 @@ mod tests {
                 None => bail!("not found tag"),
             }
         }
+
+        fn get_by_fusen_id(&self, fusen_id: FusenId) -> Result<Vec<Tag>, Error> {
+            let tags = self.tags.lock().unwrap();
+            let mut filtered_tags: Vec<Tag> = Vec::new();
+            for tag in tags.values() {
+                if tag.fusen_ids().contains(&fusen_id) {
+                    filtered_tags.push(tag.clone());
+                }
+            }
```

Comment:
for文を回すよりイテレータで処理するのが良さそうな気がします！このような書き方はどうでしょう？

```suggestion
            let filtered_tags = tags
                .values()
                .filter(|x| x.fusen_ids().contains(&fusen_id))
                .map(|x| x.clone())
                .collect::<Vec<Tag>>();
```